homeassistant/components/logbook.py

```diff
@@ -286,6 +287,32 @@ def humanify(hass, events):
                     'context_user_id': event.context.user_id
                 }
 
+            elif event.event_type == EVENT_HOMEKIT_CHANGED:
+                data = event.data
+                entity_id = data.get(ATTR_ENTITY_ID)
+                display_name = data.get('display_name')
+                message = data.get('message')
+
+                if message:
+                    action = "send command {}".format(message)
+                elif entity_id:
```

Comment:
Instead of `if / elif / else` blocks. It could be useful to construct a block beforehand and just add it:
`msg_value = " to {}".format(value) if value else ''`

`message = "send command {}{} for {}".format(service, msg_value, display_name)`

Reply:
Updated